### DISCUSSION TITLE
What legal concerns does Mozilla/the MDN incur for AI responses that lead to inaccessible code?

### DISCUSSION THREAD

**Discussion by ericwbailey** [mdn/mdn-community#413] [Announcements]
As pointed out in [this comment](https://github.com/mdn/yari/issues/9208#issuecomment-1615245134) in https://github.com/mdn/yari/issues/9208, a LLM can and does summarize content in a way that is not representative of objective truth.

When it comes to accessibility, this can mean the difference between enabling people to use web experiences and not. In the United States, this is a Civil Rights concern. It is also an issue for many international policies. This applies for government-related content, as well as the private sector (see _National Federation of the Blind v. Target Corp._, _Gil v. Winn-Dixie_, etc.).

Setting aside the concerns of if Mozilla should have added this feature, and how they went about doing so, my question is: What legal responsibilities does Mozilla incur for consciously and deliberately adding a feature that will help enable and facilitate the creation of inaccessible experiences? 

I view this as different than providing static, neutral documentation. The way LLM's interface operates in such a way that it can be interpreted as the MDN's _opinion_ about how the code should be interpreted. Furthermore, the context of "AI" output presented in the larger context of the MDN makes the output seem objective and authoritative. To me, this represents legal and reputational risk. 

While not a question I think the community call needs to answer immediately, I'd also encourage them to consider the network effects of AI-suggested inaccessible code presented as accessible being produced, and then re-incorporated into this and other code-generating and interpreting LLMs.

**Comment by caugner** [comment]
We have recorded your question as number 27.

**Comment by ericwbailey** [comment]
Is there an anticipated timeline for an answer here? 

Even a rough timeline would help with the perception that closing https://github.com/mdn/yari/issues/9208 and deferring to a synchronous meeting was an attempt to deflect, obfuscate, and suppress community questions.

**Comment by ericwbailey** [comment]
It has been a month since I previously asked, and I noticed a new Community Call has been posted to the Discord. 

I would like to reiterate my question about an anticipated timeline for an answer, as well as re-emphasize how not answering it can be perceived. I feel it is in the scope of [this comment](https://github.com/orgs/mdn/discussions/424#discussioncomment-6480596), as the question was asked prior to the session.

**Reply by ericwbailey** [reply]
While I wait, I'll be adding resources here for more context as I encounter them. I hope this helps in informing a reply:

- [ChatGPT's odds of getting code questions correct are worse than a coin flip](https://www.theregister.com/2023/08/07/chatgpt_stack_overflow_ai/)
- [Setting expectations for asking ChatGPT web accessibility questions](https://www.scottohara.me/blog/2023/01/31/ai-a11y-maybe-no.html)
- [Five accessibility bugs GitHub Copilot offers to create](https://mallonbacka.com/blog/2023/04/github-copilot-accessibility/)
- [I Made a Site Leveraging AI: How Accessible Was It?](https://dev.to/steady5063/i-made-a-site-leveraging-ai-how-accessible-was-it-4dam)
- ["AI", and the trouble with inaccessible SaaS](https://localghost.dev/blog/ai-and-the-trouble-with-inaccessible-saas/)
- [The Children Are Driving the Bus](https://adrianroselli.com/2023/09/the-children-are-driving-the-bus.html)
- [Why doesn’t AI work for producing accessible code?](https://alastairc.uk/2023/11/why-doesnt-ai-work-for-producing-accessible-code/)
- [ChatGPT is not ready to handle web accessibility remediation](https://karlgroves.com/chatgpt-is-not-ready-to-handle-web-accessibility-remediation/)

If there is a reply, I would like to see it scoped to the immediate concerns brought up in these resources. I would hope that the conversation does not steer into "we believe in the potential of this technology as it matures" territory, as I feel the immediate concerns need addressing.

**Comment by Ultrabenosaurus** [comment]
@LeoMcA @caugner @Rumyra have MDN abandoned their promise to answer all community questions?
